Aliucord/src/main/java/com/aliucord/PluginManager.java

```diff
@@ -31,7 +31,7 @@
      */
     @SuppressWarnings({"JavaReflectionMemberAccess", "unchecked"})
     public static void loadPlugin(Context context, File file) {
-        String name = file.getName().replace(".zip", "");
+        String name = file.getName().replace(".zip", "").replace(".aliu", "");
```

Comment:
Couldn't this use the regexp OR?

```suggestion
        String name = file.getName().replace("\\.(?:zip|aliu)", "");
```